samples/bluetooth/hci_vs_scan_req/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BT_CTLR_VS_SCAN_REQ_RX=y
```

Comment:
Should this be moved to the prj.conf file? 

Since the main purpose of the sample is to use this, it shouldn't be in an overlay IMO

Reply:
It was asked above to move some configs into this overlay file. Config moved and overlay removed, if it doesn't break anything in CI.

Reply:
I was only concern if the sample was to be used with other Controllers that may decide to add the feature in which case supplying overlay files seemed fair.

I have no objections as-is to have in prj.conf, and changes can  be done in the future if required.